wagtail_airtable/utils.py

```diff
@@ -138,6 +138,11 @@ def airtable_message(request, instance, message="Airtable record updated", butto
 
     Supports a custom message, custom button text, and the ability to disable buttons entirely (use case: deleting a record)
     """
+    custom_message = settings.get("WAGTAIL_AIRTABLE_PUSH_MESSAGE", '')
+
+    if custom_message and settings.get("WAGTAIL_AIRTABLE_SAVE_SYNC", False):
```

Comment:
I don't think we need to check `SAVE_SYNC` here (it also defaults to `False`, whereas in mixins.py it defaults to `True`)

Reply:
Sounds good! My thought here was that if it's enabled we don't want the default messaging as it might confuse the user, but I'll take that off! Also had it defaulting to False so we kept the default messaging.